stingray/simulator/simulator.py

```diff
@@ -311,10 +311,10 @@ def periodogram(self, lc, seg_size=None):
         -------
         ps: numpy.ndarray
             The array of normalized squared absolute values of Fourier
-            amplitudes    
+            amplitudes
 
         """
         if seg_size is None:
             seg_size = self.lc.tseg
 
-        return AveragedPowerspectrum(lc, seg_size).ps
+        return AveragedPowerspectrum(lc, seg_size).power
```

Comment:
I'm currently doing this in a separate pull request, because there was some other stuff that broke in the simulator when I refactored the `Powerspectrum` class.